src/doomstat.h

```diff
@@ -153,6 +153,7 @@ enum {
   // from PrBoom+/Eternity Engine (part of mbf21 spec)
   comp_respawn,
   comp_soul,
+  comp_666,
```

Comment:
Is it okay to change the order of the `enum` values?

Reply:
Yes, it should be fine.

Reply:
I'm a bit concerned, because in code like this the `i` will have a different meaning before and after this code change:

https://github.com/fabiangreffrath/woof/blob/ca4152176ce032946be5bd73363d489e77a72961/src/g_game.c#L4404-L4408

Reply:
Oh, right, I forgot how that stuff works. We should refactor this sometime.

I suggest making a flag specifically for the UAC_DEAD case and removing `comp_666` for now.

Reply:
And about the other entries in this enum, shouldn't we keep them in order with DSDA-Doom?

Reply:
It's already in order. There is translation system for `comp` vector in DSDA-Doom.

Reply:
I think we should do this in a separate PR. It's not so easy to add `comp_666` in our system.

Reply:
We could solve it like this back then:
https://github.com/fabiangreffrath/woof/commit/73e8bbe6f3e6f7b01f734644afb96e2a2c6e1a2f

Reply:
But then a lot of code in g_compatibility.c won't work (save/restore, get comp name, etc). We would have to add a lot of exceptions.

Reply:
Hm, right.

Reply:
> It's already in order. There is translation system for `comp` vector in DSDA-Doom.

Sorry to contradict, but Woof's and Dsda-doom's comp enums differ from the `comp_zerotags` element on and I'm not sure which translation system you mean.

Reply:
> Sorry to contradict, but Woof's and Dsda-doom's comp enums differ from the `comp_zerotags` element on and I'm not sure which translation system you mean.

I'm talking about this: https://github.com/kraflab/dsda-doom/blob/151de9390d5c542dd97bb65733d4d2c22d460a1d/prboom2/src/dsda/options.c#L332

Looks correct to me, otherwise there would be desync in MBF/MBF21 demos. Although these options are rarely used.

Reply:
Indeed, there it is. Thank you!